devenv/docker/blocks/postgres/docker-compose.yaml

```diff
@@ -1,9 +1,9 @@
   postgres:
     image: postgres:${postgres_version}
     environment:
-      POSTGRES_USER: grafana
-      POSTGRES_PASSWORD: password
-      POSTGRES_DB: grafana
+      POSTGRES_USER: grafanatest
+      POSTGRES_PASSWORD: grafanatest
+      POSTGRES_DB: grafanatest
```

Comment:
Can we please revert this change and keep it consistent with the other blocks.

Reply:
ack, I see now I missed the `postgres_tests` block... will revert this :)